generator/k8s_docs_tools/templates/component.j2

```diff
@@ -52,7 +52,7 @@ These charms are the core components of the Charmed Kubernetes release.
 ## CNI charms
 
 These are the Container Network Interface (CNI) charms supported by Charmed Kubernetes
-(Calico is the default CNI and is also included in the core charms table above).
+(calico is the default CNI and is also included in the core charms table above).
```

Comment:
why are we lowercasing this?

Reply:
I lower-cased it to match what was in the current component.md page.  Most of the template changes were made to match what was already there.